decorator.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'nameable'
+require_relative 'person'
+
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] You are required to `capitalize` the name, not make it all caps. Kindly refactor that.

  We should have `Maximilian`, not `MAXIMILIAN`.